### DISCUSSION TITLE
Reconsider logo

### DISCUSSION THREAD

**Discussion by sophie-h** [rui314/mold#432] [General]
The logo is well made. Maybe a bit too well for me :laughing: 

But, seriously, my stomach is a bit sensitive to such images. Would it be an option to re-interpret mold as the thing that you use to cast things? Just one idea of how it would be possible to solve this.

**Comment by rui314** [comment]
Got it. I'll just remove the image from the github page. Does that work for you?

**Reply by sophie-h** [reply]
I mostly see it if others or I share the link to the git repo, so it's the repo logo (#215).

PS: Mold just reduced my devel compile time by a factor of 3. It's amazing!

**Reply by rui314** [reply]
I just removed it from the logo setting for now. It is indeed a bit disgusting to see it every time someone shares a link to this repo if you don't like that image. Sorry about that!

**Reply by real-yfprojects** [reply]
Luckily this great artwork can still be found in the vcs:

<img src="https://github.com/rui314/mold/raw/6a6ae217a32ec41a05349846276f74c452187ada/docs/mold.jpg" width=150/>
